muxtools/utils/convert.py

```diff
@@ -178,3 +179,19 @@ def timedelta_from_formatted(formatted: str) -> timedelta:
     seconds = seconds + (Decimal(split[1]) * Decimal(60))
     seconds = seconds + (Decimal(split[2]))
     return timedelta(seconds=seconds.__float__())
+
+
+def sizeof_fmt(num, suffix="B"):
+    """
+    Human readable file size.
+
+    :param num:        File size in bytes/bits
+    :param suffix:     Suffix to use (change to "b" for bits)
+
+    :return:            Formatted size
+    """
+    for unit in ("", "Ki", "Mi", "Gi", "Ti", "Pi", "Ei", "Zi"):
```

Comment:
these are prefixes; `B` or `b` is the unit

Reply:
(also, why stop at `Yi`; IEC 80000-13:2025 goes up to robi (`Ri`) and quebi (`Qi`) past that)